authentication/token.md

```diff
@@ -7,7 +7,7 @@ If you already have an app started, you can simply add the following line to you
 ```js
 app.configure(authentication({
     token: {
-        secretL 'my-secret'
+        secret: 'my-secret'
```

Comment:
I think it's just this line here. @kulakowka if you could pull in the latest master and fix the conflict that would be sweet! Then we can merge the PR.